src/legacy/js/functions/_tags.js

```diff
@@ -123,3 +123,18 @@ async function getAllTopics(){
         $('#selectSubtopicsSubtopics').append(newOption)
     })
 }
+
+function validateAndSaveTags(data) {
+    if ($("#selectPrimaryTopic").length === 0 && $("#selectSubtopic").length ===0) {
```

Comment:
Can we have an extra space? 
```suggestion
    if ($("#selectPrimaryTopic").length === 0 && $("#selectSubtopic").length === 0) {
```